client/src/Components/Layout.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Footer from './Footer';
+import Navbar from './Navbar';
+
+const Layout = ({ children }) => {
+  return (
+    <>
+      <Navbar />
+      {children}
+      <Footer />
+    </>
```

Comment:
Change the fragment to a div. It's best to set this div as having a maxWidth. You'll also want to make sure to center the divs children (Navbar, children, Footer).

Wrap the {children} prop in a main tag for better visibility in the html.